Services/DataX.Flow/DataX.Flow.SchemaInference/Blob/BlobMessageBus.cs

```diff
@@ -34,6 +34,7 @@ public BlobMessageBus(List<FlowGuiInputBatchInput> batchInputs, ILogger logger)
         public async Task<EventsData> GetSampleEvents(int seconds)
         {
             EventsData eventsData = new EventsData();
+            const int numberOfDocumentsToRead = 500;
```

Comment:
does this affect streaming as well?  can this be a parameter to the function

Reply:
No this code path is only for blob input.